### PR TITLE
Create list command

### PR DESCRIPTION
**Pull request by wjayee**
Changes made:
- Add List Command
- Refactor some classes to switch names away from Person

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ListAnimalCommand.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import seedu.address.model.AnimalModel;
+
+/**
+ * Lists all animals in the catalog to the user.
+ */
+public class ListAnimalCommand extends AnimalCommand {
+
+    public static final String COMMAND_WORD = "list";
+
+    public static final String MESSAGE_SUCCESS = "Listed all persons";
```
Comment (Blizzeracz) [inline src/main/java/seedu/address/logic/commands/ListAnimalCommand.java:15]:
```suggestion
    public static final String MESSAGE_SUCCESS = "Listed all animals";
```

Just a minor mistake!

File: src/main/java/seedu/address/model/AnimalModel.java
Comment (Blizzeracz) [inline src/main/java/seedu/address/model/AnimalModel.java:1]: Thanks for editing the changes to Animal Catalog!

**Review by Blizzeracz — APPROVED** [review]
Overall, looks good to merge!

File: src/main/java/seedu/address/logic/commands/ListAnimalCommand.java (hunk 1 above)
Comment (wjayee) [reply src/main/java/seedu/address/logic/commands/ListAnimalCommand.java]: Oh right thanks!

**Pull request merged**